pinakes/main/approval/models.py

```diff
@@ -184,10 +189,96 @@ class Meta:
             ),
         ]
 
+    def move_internal_sequence(self, delta):
```

Comment:
Can you please add some explanation of how this works and how do you use Decimal field for `Workflow.internal_sequence`.